### PR TITLE
[luci-interpreter] Enable float SquaredDifference

### PR DESCRIPTION
**Pull request by seanshpark**
This will enable float of SquaredDifference.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #6362

File: compiler/luci-interpreter/src/kernels/SquaredDifference.test.cpp [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
```
Comment (jinevening) [inline compiler/luci-interpreter/src/kernels/SquaredDifference.test.cpp:2]:
```suggestion
 * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
```

File: compiler/luci-interpreter/src/kernels/SquaredDifference.test.cpp [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  Tensor output_tensor = makeOutputTensor(DataType::FLOAT32);
+
+  SquaredDifference kernel(&input_tensor1, &input_tensor2, &output_tensor);
+  kernel.configure();
+  kernel.execute();
+
+  std::vector<float> ref_output_data{4.0, 0.0, 4.0, 1.0, 1.0, 0.0001};
+  EXPECT_THAT(extractTensorData<float>(output_tensor), FloatArrayNear(ref_output_data));
+}
+
```
Comment (jinevening) [inline compiler/luci-interpreter/src/kernels/SquaredDifference.test.cpp:46]: (optional) It would be great to add a test for broadcasting.

**Review by jinevening — APPROVED** [review]

**Review by jinevening — APPROVED** [review]
LGTM

**Pull request merged**